packages/main/src/ColorPalette.js

```diff
@@ -24,6 +28,16 @@ const metadata = {
 	tag: "ui5-color-palette",
 	managedSlots: true,
 	properties: /** @lends sap.ui.webcomponents.main.ColorPalette.prototype */ {
+		/**
+		 * Defines whether the user can choose a custom color from a color picker
```

Comment:
Also add a note to remind users to import the required moreColors feature module.